src/components/KMultiselect/KMultiselect.vue

```diff
@@ -1158,7 +1189,7 @@ onBeforeUnmount(() => {
       // Firefox workaround
       // since :has() selector isn't supported in Firefox be default
       .k-multiselect-list ~ .k-multiselect-dropdown-footer-sticky {
-        bottom: 0;
+        bottom: var(--kui-space-0, $kui-space-0);
```

Comment:
issue: don't use a space token for positioning

Reply:
Fixed